packages/graphql/src/server.js

```diff
@@ -81,7 +84,48 @@ if (typeof API_KEY !== 'undefined') {
   }
 }
 
+const app = express()
+app.use(bundle)
+
+// Extract functions name and calling args.
+const gqlQueryExtractRe = /query *(.*) *\{/
+
+// Extract at most 32 chars to avoid spamming the logs.
+const queryMaxExtractSize = 32
+
+// Utility method to extract a Graphl query from the POST body.
+function extractGqlQuery(req) {
+  if (!req.body || !req.body.query) {
+    return ''
+  }
+  const query = req.body.query
+  const matches = query.match(gqlQueryExtractRe)
+  if (matches && matches.length) {
+    return matches[1].slice(0, queryMaxExtractSize)
+  }
+  // Fallback to extract the first 32 chars after the query string.
+  console.log('FALLING BACK')
```

Comment:
nit: please remove this log (I had it in disco too but removed it since then...)